manifests/manifest-chrome.json

```diff
@@ -41,32 +41,36 @@
         ]
     },
     "background": {
-        "scripts": [
-            "dist/background.js"
-        ],
-        "persistent": true
+        "service_worker": "dist/background.js"
     },
     "sandbox": {
         "pages": [
             "view/argon.html"
-        ],
-        "content_security_policy": "sandbox allow-scripts; script-src 'self' 'unsafe-eval';"
+        ]
     },
     "permissions": [
         "activeTab",
         "storage",
-        "identity"
+        "identity",
+        "alarms",
+        "scripting"
     ],
     "optional_permissions": [
         "clipboardWrite",
-        "contextMenus",
-        "https://www.google.com/",
-        "https://*.dropboxapi.com/*",
-        "https://www.googleapis.com/*",
-        "https://accounts.google.com/o/oauth2/revoke",
-        "https://graph.microsoft.com/me/*",
-        "https://login.microsoftonline.com/common/oauth2/v2.0/token"
+        "contextMenus"
+    ],
+    "optional_host_permissions": [
+      "https://www.google.com/",
+      "https://*.dropboxapi.com/*",
+      "https://www.googleapis.com/*",
+      "https://accounts.google.com/o/oauth2/revoke",
+      "https://graph.microsoft.com/me/*",
+      "https://login.microsoftonline.com/common/oauth2/v2.0/token"
     ],
     "offline_enabled": true,
```

Comment:
I can't find any information that suggests that `offline_enabled` is still supported by Manifest V3. Compare these pages:
https://developer.chrome.com/docs/extensions/mv2/manifest/
https://developer.chrome.com/docs/extensions/mv3/manifest/

[The page for this property on MDN Web Docs](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/offline_enabled) also lists the supported manifest version as 2 only.